Modules/Devices/src/Devices.Domain/Aggregates/PushNotifications/Handles/PnsHandle.cs

```diff
@@ -19,17 +19,17 @@ protected PnsHandle(PushNotificationPlatform platform, string value)
         {
             case PushNotificationPlatform.Fcm:
             {
-                var handle = FcmHandle.Parse(value);
-                return handle.IsSuccess
-                    ? Result.Success<PnsHandle, DomainError>(handle.Value)
-                    : Result.Failure<PnsHandle, DomainError>(DomainErrors.InvalidPnsHandleParse($"Value '{value}' could not be parsed for platform '{platform}'"));
+                var parseHandleResult = FcmHandle.Parse(value);
```

Comment:
I'm not sure if C# supports this, but can you try to return the parseHandleResult directly?